systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaClientsResource.java

```diff
@@ -210,24 +165,24 @@ private static PodSpec createClientSpec(boolean tlsListener, String kafkaClients
 
                     containerBuilder
                         .addNewVolumeMount()
-                            .withName(clusterCaSecretVolumeName)
-                            .withMountPath(caSecretMountPoint)
+                        .withName(clusterCaSecretVolumeName)
+                        .withMountPath(caSecretMountPoint)
                         .endVolumeMount()
                         .addNewEnv().withName("PRODUCER_TLS" + envVariablesSuffix).withValue("TRUE").endEnv()
                         .addNewEnv().withName("CONSUMER_TLS" + envVariablesSuffix).withValue("TRUE").endEnv()
                         .addNewEnv().withName("CA_LOCATION" + envVariablesSuffix).withValue(caSecretMountPoint).endEnv()
-                        .addNewEnv().withName("TRUSTSTORE_LOCATION" + envVariablesSuffix).withValue("/tmp/"  + kafkaUserName + "-truststore.p12").endEnv();
+                        .addNewEnv().withName("TRUSTSTORE_LOCATION" + envVariablesSuffix).withValue("/tmp/" + kafkaUserName + "-truststore.p12").endEnv();
 
                     if (tlsUser) {
                         containerBuilder.addNewEnv().withName("KEYSTORE_LOCATION" + envVariablesSuffix).withValue("/tmp/" + kafkaUserName + "-keystore.p12").endEnv();
                     }
 
                     podSpecBuilder
                         .addNewVolume()
-                            .withName(clusterCaSecretVolumeName)
-                            .withNewSecret()
-                                .withSecretName(clusterCaSecretName)
-                            .endSecret()
+                        .withName(clusterCaSecretVolumeName)
+                        .withNewSecret()
+                        .withSecretName(clusterCaSecretName)
+                        .endSecret()
```

Comment:
Why removing all the formatting which made it easier to read when you already had it (here and several other places)?

Reply:
Yes sorry, maybe problem when I was moving the code.